### PR TITLE
add control_type table

### LINKED ISSUE #418: List of the supported control types is not complete
http://docs.workato.com/developing-connectors/sdk/object-definition.html#components

We support more control types, at least these:
- `schema-designer`
- `static-list`
- `schema-builder`
- `plain-text`
- `plain-text-area`
- `radio`
- `multiselect`
- `code`
- `date`

### PR DESCRIPTION
**Pull request by eeshansim**
fixes #418

### CODE REVIEW

File: developing-connectors/sdk/object-definition.md [hunk 1]
```diff
@@ -129,6 +117,124 @@ object_definitions: {
...
+    <tr>
+      <th>Control type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>text</td>
+      <td>
+        Simple text field input field with formula mode option.<br>
```
Comment (jandonyada) [inline developing-connectors/sdk/object-definition.md]: Simple text ~~field~~ input field

File: developing-connectors/sdk/object-definition.md [hunk 2]
```diff
@@ -129,6 +117,124 @@ object_definitions: {
...
+      <td>text-area</td>
+      <td>
+        Long text input field with formula mode option.<br>
+        <img src="/assets/images/sdk/text-area.png" alt="text-area control type">
+      </td>
+    </tr>
+    <tr>
+      <td>plain-text</td>
+      <td>
+        Simple text are field input field without formula mode option.<br>
```
Comment (jandonyada) [inline developing-connectors/sdk/object-definition.md]: Simple text ~~are field~~ input field

File: developing-connectors/sdk/object-definition.md [hunk 3]
```diff
@@ -129,6 +117,124 @@ object_definitions: {
...
+    </tr>
+    <tr>
+      <td>select</td>
+      <td>
+        Control type to provide a predefined list of values to choose from. Make sure to include the <code>pick_list</code> property.<br>
+        <img src="/assets/images/sdk/select.png" alt="select control type">
+      </td>
+    </tr>
+    <tr>
+      <td>checkbox</td>
```
Comment (jandonyada) [inline developing-connectors/sdk/object-definition.md]: This is probably how it's being implemented, but it's really weird that this control type is called 'Checkbox' when it doesn't look anything like it..

Comment (eeshansim) [reply developing-connectors/sdk/object-definition.md]: Yea it's a little off

**Pull request merged**